iamy.go

```diff
@@ -68,8 +69,9 @@ func main() {
 
 	case pull.FullCommand():
 		PullCommand(ui, PullCommandInput{
-			Dir:       *pullDir,
-			CanDelete: *canDelete,
+			Dir:                  *pullDir,
+			CanDelete:            *canDelete,
+			HeuristicCfnMatching: !*lookupCfn,
```

Comment:
I'm finding it hard to figure out if this is on by default or not.
If you set this flag won't it disable it?

Reply:
passing `--accurate-cfn` will disable the faster heuristic matching.